R/cdfPlot.R

```diff
@@ -171,117 +173,154 @@ cdfPlot <- function(contrastDF,
     subsetRows <- sum(contrastDF[[y]] <= pvalMax)
     contrastDFsubset <- contrastDF[1:subsetRows,]
 
-    # Plot subset percent of the data for the main plot
-    cdfMain <- ggplot(contrastDFsubset, aes_string(x = x, y = y)) +
-        aes(shape = group, size = group, color = group, fill = group, order = order) +
-        # Scale lines tell it to use the actual values, not treat them as factors
-        scale_shape_manual(name = "Group", guide = "legend", labels = ssc$group, values = ssc$symbolShape) +
-        scale_size_manual(name = "Group", guide = "legend", labels = ssc$group, values = ssc$symbolSize) +
-        scale_color_manual(name = "Group", guide = "legend", labels = ssc$group, values = ssc$symbolColor) +
-        scale_fill_manual(name = "Group", guide = "legend", labels = ssc$group, values = ssc$symbolFill) +
-        geom_point(alpha = alpha)
-
-    # Optional Decorations
-    if (!is.null(referenceLine)) {
-        cdfMain <- cdfMain +
-            geom_hline(yintercept = pThreshold, color = referenceLine,
-                       size = refLineThickness, alpha = 0.5)
-    }
+    if (plotType == "canvasXpress") {
+        ## Create the canvasXpress df and var annotation
+        # Main plot
```

Comment:
comments seem incongruent, what is happening here - creating the df/var or the main plot ?  Comments should add to the user's reading of the code when it isn't clear

Reply:
I have updated the comment.